### ISSUE TITLE
A11y_code.visualstudio.com_API Working With Extensions_Continuous Integration_Keyboard: Tooltip is not visible on "Delete button" for keyboard users.

### ISSUE THREAD

**Issue by RohitJadhav-TCS** [microsoft/vscode-docs#3705]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-manai%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_Code.visualstudio.com_Benchmark;#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark;#ChromiumEdge;#Chrome;#DesktopWeb;#MAS1.1.1;#Win10;#Visual Studio Code;#Keyboard; #CAITestPass_May2020;#FTP;#STP;

**Environment Details**:
Application Name: code.visualstudio.com
#URL:https://code.visualstudio.com
Windows Version: Windows10
Browser version: Microsoft Edge Version 84.0.516.1 (Official build) dev (64-bit)

**Repro Steps**:
1. Launch the application URL :https://code.visualstudio.com
2. Navigate to "API" link.
3. From left navigation, select "Working With Extensions".
4. Navigate to "Continuous Integration" link.
5. Navigate to Search box then select any item from recent search entry and navigate to delete button using tab key.

**Actual**:
Tooltip is not visible on "Delete button" for keyboard users.

**Similar issue observed in through out the application where this search box is present.** 

**Expected**:
Tooltip should be visible when keyboard focus goes on that control.

**Attachment link for Reference**:
![3705_A11y_code visualstudio com_API Working With Extensions_Continuous Integration_Keyboard Tooltip is not visible on Delete button for keyboard users](https://user-images.githubusercontent.com/52035395/83134730-6681a380-a102-11ea-9ccf-6f3492309ff4.gif)

**Comment by RohitJadhav-TCS** [comment]
Reusing the bug

**Comment by sbatten** [comment]
the delete item button is a native browser feature. this is an upstream issue.

**Comment by gregvanl** [comment]
Assigning back to issue author per test pass guidance

**Comment by kapilvaishna** [comment]
For universal icon button if tool tip is not accessible with keyboard it's not a bug so closing the bug.
#Closed:
